src/easylink/cli.py

```diff
@@ -201,16 +201,20 @@ def run(
     main = handle_exceptions(
         func=runner.main, exceptions_logger=logger, with_debugger=with_debugger
     )
-    main(
-        command="run",
-        pipeline_specification=pipeline_specification,
-        input_data=input_data,
-        computing_environment=computing_environment,
-        results_dir=results_dir,
-        images_dir=images,
-        schema_name=schema,
-    )
-    logger.info("*** FINISHED ***")
+    try:
+        main(
+            command="run",
+            pipeline_specification=pipeline_specification,
+            input_data=input_data,
+            computing_environment=computing_environment,
+            results_dir=results_dir,
+            images_dir=images,
+            schema_name=schema,
+        )
+    except SystemExit:
```

Comment:
I'd like to discuss this - I don't really see why it was necessary just to see the `*** FINISHED ***` message at the very end. Snakemake always systemexits (even when successful), so I don't why this change was needed just b/c we wrapped snake_main with the filtering function.